jobserver/admin.py

```diff
@@ -54,3 +54,31 @@ class Meta:
 @admin.register(Project)
 class ProjectAdmin(admin.ModelAdmin):
     form = ProjectForm
+    fieldsets = [
+        [
+            None,
+            {
+                "fields": [
+                    "org",
+                    "name",
+                    "display_name",
+                    "email",
+                    "project_lead",
+                    "proposed_start_date",
+                    "proposed_duration",
+                ],
+            },
+        ],
+        [
+            "Technical Governance",
+            {"fields": ["governance_approval_notes", "has_governance_approval"]},
+        ],
+        [
+            "Technical Governance",
```

Comment:
Should this have the same heading as the section above?

Reply:
It should not… copypasta gone wrong 😬